ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -1,400 +1,373 @@
-    if (mBannerRequest == null) {
-      //Adapter does not support multiple Banner instances playing for same placement except for Refresh
-      mMediationBannerListener
-          .onAdFailedToLoad(VungleInterstitialAdapter.this, AdRequest.ERROR_CODE_INVALID_REQUEST);
-      return;
-    }
-
-    mBannerRequest.setAdLayout(adLayout);
-    mBannerRequest.setVungleListener(mVungleBannerListener);
-    mBannerRequest.requestBannerAd(context, config.getAppId());
-  }
-
-  private VungleListener mVungleBannerListener = new VungleListener() {
-    @Override
-    void onAdEnd(String placement, boolean wasSuccessfulView, boolean wasCallToActionClicked) {
-      if (mMediationBannerListener != null) {
-        if (wasCallToActionClicked) {
-          // Only the call to action button is clickable for Vungle ads. So the
-          // wasCallToActionClicked can be used for tracking clicks.
-          mMediationBannerListener.onAdClicked(VungleInterstitialAdapter.this);
-          mMediationBannerListener.onAdOpened(VungleInterstitialAdapter.this);
-          mMediationBannerListener.onAdClosed(VungleInterstitialAdapter.this);
-        }
-      }
-    }
-
-    @Override
-    void onAdAvailable() {
-      if (mMediationBannerListener != null) {
-        mMediationBannerListener.onAdLoaded(VungleInterstitialAdapter.this);
-      }
-    }
-
-    @Override
-    void onAdStart(String placement) {
-      //let's load it again to mimic auto-cache, don't care about errors
-      if (mBannerRequest != null) {
-        mBannerRequest.preCache();
-      }
-    }
-
-    @Override
-    void onAdFail(String placement) {
-      Log.w(TAG, "Ad playback error Placement: " + placement + ";" + mBannerRequest);
-    }
-
-    @Override
-    void onAdFailedToLoad(int errorCode) {
-      Log.w(TAG, "Failed to load ad from Vungle: " + errorCode + ";" + mBannerRequest);
-      if (mMediationBannerListener != null) {
-        mMediationBannerListener.onAdFailedToLoad(VungleInterstitialAdapter.this, errorCode);
-      }
-    }
-  };
-
-  @Override
-  public View getBannerView() {
-    Log.d(TAG, "getBannerView # instance: " + hashCode());
-    return adLayout;
-  }
-
-  private boolean hasBannerSizeAd(Context context, AdSize adSize, AdConfig adConfig) {
-    ArrayList<AdConfig.AdSize> potentials = new ArrayList<>(4);
-    potentials.add(0, BANNER_SHORT);
-    potentials.add(1, BANNER);
-    potentials.add(2, BANNER_LEADERBOARD);
-    potentials.add(3, VUNGLE_MREC);
-
-    Log.i(TAG, "Potential ad sizes: " + potentials.toString());
-    AdConfig.AdSize closestSize = findClosestSize(context, adSize, potentials);
-    if (closestSize == null) {
-      Log.i(TAG, "Not found closest ad size: " + adSize);
-      return false;
-    }
-    Log.i(TAG,
-        "Found closest ad size: " + closestSize.toString() + " for request ad size:" + adSize);
-
-    adConfig.setAdSize(closestSize);
-
-    return true;
-  }
-
-  // Copied some code from FB adapter:

```

Comment:
@HuFangshuai @jiancchen Seems like AdMob. removed this part of logic, will it have any bad effect on our logic to map to our SDK sizes , will we have to update our Test cases or Requirements ?

Reply:
I checked MediationUtils#findClosestSize the radio for MIN_WIDTH_RATIO and MIN_HEIGHT_RATIO not changed, I added one unit test in test app and verified, so no need to update test cases.